src/Rings/mpoly-ideals.jl

```diff
@@ -898,31 +907,26 @@ Multivariate polynomial ring in 2 variables over number field graded by
   y -> [1]
 ```
 """
-@attr Vector{Tuple{typeof(I), typeof(I), ideal_type(change_base_ring(rationals_as_number_field()[1], base_ring(I))[1]), Int64}} function absolute_primary_decomposition(I::MPolyIdeal{<:MPolyRingElem{QQFieldElem}})
```

Comment:
all of the below has been changed by https://github.com/oscar-system/Oscar.jl/pull/4071. Why is it changed back?